### ISSUE TITLE
Release Notes - PR release notes only showing most recent version information

### ISSUE THREAD

**Issue by setchy** [renovatebot/renovate#10955]
**How are you running Renovate?**

- [x] WhiteSource Renovate hosted app on github.com
- [ ] Self hosted

If using the hosted app, please skip to the next section.
Otherwise, if self-hosted, please complete the following:

Please select which platform you are using:

- [ ] Azure DevOps (dev.azure.com)
- [ ] Azure DevOps Server
- [x] Bitbucket Cloud (bitbucket.org)
- [ ] Bitbucket Server
- [ ] Gitea
- [x] github.com
- [ ] GitHub Enterprise Server
- [ ] gitlab.com
- [ ] GitLab self-hosted

Renovate version: ...  @latest 

**Describe the bug**
When testing renovate against [apollographql/federation-jvm](https://github.com/apollographql/federation-jvm), it correctly identified that the `pom.xml` should be updated from version 0.5.0 to 0.6.4.

However, the release notes added to the PR only have the most recent entry for v0.6.4 and has not added any previous ones (see `federation-jvm` [RELEASE_NOTES.md](https://github.com/apollographql/federation-jvm/blob/master/RELEASE_NOTES.md))

Example renovate PR to recreate the scenario - https://github.com/setchy/renovate-bot-testbed/pull/1

Interestingly, for #10954 it is showing all release notes from the `CHANGELOG.md`
...

**Relevant debug logs**

<details><summary>Click me to see logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [x] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

...

**Comment by setchy** [comment]
**TLDR**
After some local debugging this seems to due to the shutdown of jcenter.bintray.com and certain jars not being available on repo.maven.apache.org.  


**Details**
When renovate checks the datasource (maven) for releases of `com.apollographql.federation:federation-graphql-java-support` since v0.5.0, it correctly finds the 5 available from [repo.maven.apache.org](https://repo.maven.apache.org/maven2/com/apollographql/federation/federation-graphql-java-support/).  They are:
* 0.6.3-RC1
* 0.6.3
* 0.6.4
* 0.6.5
* 0.7.0.

And the release notes for these are showing within the PR body.

Versions of `com.apollographql.federation:federation-graphql-java-support` between 0.5.0 and 0.6.3-RC1 were published only to jcenter.bintray.com.  These are:
* 0.6.2
* 0.6.1
* 0.6.0


Although you can see them on https://mvnrepository.com/artifact/com.apollographql.federation/federation-graphql-java-support?repo=jcenter, they are not discoverable via https://mvnrepository.com/artifact/com.apollographql.federation/federation-graphql-java-support?repo=jcenter and as such are omitted.
